muon-detector-tools/muon_fit/muon_fit.py

```diff
@@ -0,0 +1,241 @@
+import glob
+import numpy as np
+from astropy.io import fits
+from scipy.stats import norm
+import matplotlib.pyplot as plt
+from muon_fit.muon_sample import muon_sample
+
+
+def muon_fit(data_dir="", mw0=0.62, mw1=0.73, gplot=None, hplot=None, ps=None, crplot=None, verbose=None):
+    # Initial parameters
+    bin_size = 0.025
+
+    # Get image list
+    imlist = sorted(glob.glob(f'{data_dir}/img*_00.fits'))
```

Comment:
Does this mean the user has to provide a certain name prepended or appended to their fits files?

Reply:
We should probably also break apart this muon_fit function
- New function to simple load the data 
- Second function can simply do the processing